serverless/telegram-handler/.env-example

```diff
@@ -1,5 +1,7 @@
 # Required
 DB_URI=postgres://localhost:5432/postmangovsg_dev
+TELEGRAM_BOT_CONTACT_US_URL=https://go.gov.sg/postman-contact-us-recipient
+TELEGRAM_BOT_GUIDE_URL=https://guide.postman.gov.sg/faqs/faq-recipients
```

Comment:
I know this is an example file, and I presume we'll be using this value as well when we deploy. Will it make sense for this to be a go link as well? In the event that we change our guide.

Reply:
Ah good point, I initially went with this because it's consistent with frontend (Go links for forms, normal links for the rest) but the use case here is a bit different. Will make the changes.